docs/modules/utils/combine_docs_examples/vectorstores.ipynb

```diff
@@ -16,7 +16,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 37,
```

Comment:
Why 37?

Reply:
While I was testing this using OpenAI embeddings. Due to some issue with their server, I didn't get any result. So, I ran it multiple times which caused this count to increase. It is just metadata.

Reply:
Ok to reduce changes in PR, can we reset to 1?